guides/0113-random_source.md

```diff
@@ -0,0 +1,69 @@
+# random_source
+
+* type: producer
+* output: either integer, real or boolean messages
+
+Producer block that generates random messages of a specific type with a fixed key.
+
+# Properties
+
+* `key`: A unicode string that will be used as key in the generated messages. (required, string)
+* `type` The type of data that will be generated. Supported types are `integer`, `real` and `boolean`. (required, string)
+* `delay_ms`: If present, the block will wait `delay_ms` before emitting a new sample. (optional, integer)
+
+When type is `"integer"`
+* `min`: The minimum generated value, default is 0. (optional, numeric)
+* `max`: The maximum generated value, default is 100. (optional, numeric)
+
+When type is `"real"`
+* `min`: The minimum generated value, default is 0. (optional, numeric)
+* `max`: The maximum generated value, default is 1. (optional, numeric)
+
+When type is `"boolean"`
+* `p`: The probability of the generator returning `true`. Defaults to `0.5`
+  The value must be `>= 0` and `<= 1`. (optional, numeric)
+
+## `key`
+
+The key to be used in generated messages. (e.g. `"my_example_key"`).
+
+## `type`
+
+The type of the data contained in generated messages (e.g. `"integer"`).
+
+## `min`
+
+The minimum generated value. (e.g. `0`)
+
+## `max`
+
+The maxiimum generated value. (e.g. `100`)
+
+## `p`
+
+The probability of the generator returning `true`. (e.g. `0.5`)
+
+## `delay_ms`
+
+The delay between generated messages in milliseconds. For example use `5000` for 5 seconds.
+
+# Output message
+
+* `key`: contains the `key` specified as parameter.
+* `data`: contains the generated value.
+* `type`: the `type` specified as parameter.
+* `timestamp` contains the timestamp (UTC).
```

Comment:
I think that timestamp is always the same, so we should copy and paste from other existing blocks.